clients/spring-boot-starter-camunda-sdk/src/test/java/io/camunda/spring/client/properties/CamundaClientModesTest.java

```diff
@@ -76,30 +76,6 @@ void shouldLoadDefaultsSelfManaged() {
       assertThat(properties.getMode()).isEqualTo(ClientMode.selfManaged);
       assertThat(properties.getGrpcAddress().toString()).isEqualTo("http://localhost:26500");
       assertThat(properties.getRestAddress().toString()).isEqualTo("http://localhost:8088");
-      assertThat(properties.getAuth().getTokenUrl())
```

Comment:
🔧   seems worth asserting the auth mode default (`none` right?)? probably also worth for the saas test.

Reply:
There is another test in place asserting this already. Or do you mean asserting on the combination of `self-managed` mode and the implicit behaviour?

Reply:
exactly the effect of the mode on the implicit behavior

Reply:
I will add more tests around this

Reply:
More tests added